execution/stagedsync/exec3.go

```diff
@@ -231,7 +231,7 @@ func ExecV3(ctx context.Context,
 	defer resetCommitmentGauges(ctx)
 	defer resetDomainGauges(ctx)
 
-	stepsInDb := rawdbhelpers.IdxStepsCountV3(applyTx)
+	stepsInDb := rawdbhelpers.IdxStepsCountV3(applyTx, agg.StepSize())
```

Comment:
better get it by `applyTx.Debug().StepSize()` - because better don't use `agg` object directly. but no problem.